src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/AbstractRating.java

```diff
@@ -102,19 +113,18 @@ public boolean equals(Object o) {
       return false;
     }
     AbstractRating that = (AbstractRating) o;
-    return Objects.equals(score, that.score) &&
-        version == that.version;
+    return Objects.equals(score, that.score) && version == that.version;
```

Comment:
Just a question, why don't we use Objects.equals(version, that.version) here?

Reply:
`version` is a enum, the simplest way to compare them is `==`.

Reply:
Aah missed that it's an enum :D thanks for the clarification!